include/zephyr/sys/list_gen.h

```diff
@@ -234,6 +234,39 @@
 		return false;						 \
 	}
 
+/**
+ * @brief Macro to check if a node is linked in a generic list
+ * 
+ * @param __lname The list name
+ * @param __nname The node name
+ * @param __nname The next node name
```

Comment:
/**
 * @brief Macro to check if a node is linked in a generic list
 * 
 * @param __lname The list name
 * @param __nname The node name
 * @param __nname The previous node name